src/app/app.component.html

```diff
@@ -40,5 +40,6 @@
   -->
   </div>
 </nav>
-
+<!-- just for testing purposes -->
+<app-wedding></app-wedding>
```

Comment:
la pertinence ???

Reply:
<!-- just for testing purposes -->

Pour ne pas me demander le path pour voir le formulaire.